### PR TITLE
Daniel Kill's Contributor Card

### PR DESCRIPTION
**Pull request by Daniel-Kill**
Added my card to the index.html.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *Daniel Kill* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Daniel Kill</p>
+          <p class="contact">
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/daniel-kill-57014630a/" target="_blank">Your handle</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:129]:
```suggestion
            <a href="https://www.linkedin.com/in/daniel-kill-57014630a/" target="_blank">daniel-kill-57014630a</a>
```

File: index.html [hunk 2]
```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
...
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/Daniel-Kill" target="_blank">Your handle</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:131]:
```suggestion
            <a href="https://github.com/Daniel-Kill" target="_blank">Daniel-Kill</a>
```

**Review by StackOverflowIsBetterThanAnyAI — COMMENTED** [review]
Please adjust the displayed name.

**Comment by Daniel-Kill** [comment]
> Please adjust the displayed name.

I'm confused, I"m sorry which part ?
My name is really Daniel Kill
if that is what you mean

**Comment by CandidDeer** [comment]
Closing since duplicate PR.

**Pull request closed without merging**